src/amp-story-player/amp-story-entry-point/amp-story-entry-point.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpStoryPlayerManager} from '../amp-story-player-manager';
+
+self.onload = () => {
+  const manager = new AmpStoryPlayerManager(self);
```

Comment:
LGTM but maybe rename it to something generic if it will manage the lifecycle of all of our components on the publisher page

Reply:
Done.